docs/installation/considerations.mdx

```diff
@@ -0,0 +1,62 @@
+---
+sidebar_position: 2
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Vorüberlegungen
+
+Du bist fachlich bewandert und weist auf welcher Hardware evcc installiert und welche Wallbox, Solaranlage und ggf. Batterie eingebunden werden soll? Dann überspringe diese Seite und gehe direkt zur Installation.
+
+Wenn das Laden mit Solarstrom noch neu für dich ist, findest du im Folgenden einige Hinweise wie der Start mit evcc aussehen könnte.
+
+## Ausgangslage
+
+Ist evcc die richtige Lösung zum Überschussladen für mich? Ich habe eine Solaranlage und ein E-Auto und möchte das mal probieren. Wo fange ich an?
+
+Eine Solaranlage ist besonderes wirtschaftlich, wenn ich den selbst erzeugten Strom auch selbst verbrauche. Dazu muss der Strom genau dann verbraucht werden, wenn er erzeugt wird, also die Sonne scheint. Die einfachste Lösung: Ich schließe das Auto immer dann zum Laden an, wenn die Sonne scheint und keine Wolke in Sicht ist. Leider haben wir Wolken oder wir verbrauchen den Strom der Solaranlage gerade für die Waschmaschine. Wir müssen die Ladesteuerung daher automatisieren - und genau das macht evcc.
+
+Für die Automatisierung müssen wir wissen, wie hoch der aktuelle Stromüberschuss ist, den wir nicht selber verbrauchen und daher in das Stromnetz (engl. Grid) einspeisen. Diese Information bekommen wir vom Stromzähler (engl. Meter). Das kann der Stromzähler des Energieversorgers (z.B. Stadtwerke) sein oder mit der Solaranlage wurde ein entsprechender zusätzlicher Zähler verbaut. Letzteres ist meist dann der Fall, wenn die Solaranlage mit einem Speicher ausgestattet ist. 
```

Comment:
Stromzähler des **Messstellenbetreibers**, der Energieversorger hat damit nichts am Hut.

Reply:
Korrigiert.

Reply:
@uwen70 du musst nicht kommentieren. Wenn Du Updates pushst zeigt GH die Kommentare automatisch als „veraltet“ an.

Reply:
@andig, @premultiply:  Wem obliegt in diesen Fällen mit 'Resolve conversation' die Anmerkung abzuschließen?

Reply:
In erster Line dir.
Dient der Orientierung wo ggf. noch unbearbeitete bzw. unbemerkte Kommentare sind.

Reply:
Ich mach das normalerweise als Reviewer wenn ich sehe dass es erledigt ist. Jeder neue Commit löst ja wieder eine Meldung aus und man muss sich den Stand dann anschauen.